buffer/src/test/java/io/netty/buffer/PooledByteBufAllocatorTest.java

```diff
@@ -661,4 +665,27 @@ public void testCacheWorksForNormalAllocations() {
         assertEquals(3, allocator.metric().directArenas().get(0).numNormalAllocations());
         buffer.release();
     }
+
+    @Test
+    public void testNormalPoolSubpageRelease() {
+        // 16 < elemSize <= 7168 or 8192 < elemSize <= 28672, 1 < subpage.maxNumElems <= 256
+        // 7168 <= elemSize <= 8192, subpage.maxNumElems == 1
+        int elemSize = 1024;
```

Comment:
If you make `elemSize = 8192` then this test will fail when your `PoolSubpage` changes are commented out.

```suggestion
        int elemSize = 8192;
```

Reply:
yes, is there a problem? @chrisvest

Reply:
If my change is commented and the unit test fails, doesn't that mean the change is correct

Reply:
Yes. My point is: when your `PoolSubpage` changes are commented out, the test does _not_ fail when the `elemSize` is 1024, but it _does_ fail when `elemSize` is 8192, which is why I'm suggesting making this change.

With your `PoolSubpage` changes in place, the test of course pass for either `elemSize`.

Reply:
oh, I see what you mean. I did this before, but you said that my changes were not covered by the unit tests